timelock-api/src/main/conjure/timelock-api.yml

```diff
@@ -20,6 +20,14 @@ types:
       base-type: any
       external:
         java: com.palantir.lock.v2.Lease
+    CommitTimestampsWithWatches:
+      base-type: any
+      external:
+        java: com.palantir.lock.watch.CommitTimestampsWithWatches
+    Long:
```

Comment:
I believe you're going to need to rebase, @j-baker got a similar thing in with #4622